qoffer-domain/src/main/groovy/life/qbic/portal/portlet/offers/update/UpdateOfferOutput.groovy

```diff
@@ -0,0 +1,33 @@
+package life.qbic.portal.portlet.offers.update
+
+import life.qbic.datamodel.dtos.business.Offer
+import life.qbic.portal.portlet.UseCaseFailure
+
+/**
+ * Output interface for the {@link life.qbic.portal.portlet.offers.update.UpdateOffer} use case
+ *
+ * @since: 1.0.0
+ */
+interface UpdateOfferOutput extends UseCaseFailure {
+
+    /**
+     * Confirms saving of the updated offer by providing
+     * the updated offer information including the assigned identifier.
+     *
+     * @param createdOffer information about the updated offer packaged in an
+     * {@link life.qbic.datamodel.dtos.business.Offer} data-transfer object
+     * @since 1.0.0
+     */
+    void updatedOffer(Offer createdOffer)
```

Comment:
here we had a discussion with @jenniferboedker recently, about being more explicit in the naming, that methods are called based on events.

```suggestion
    void onUpdatedOffer(Offer updatedOffer)
```

or

```suggestion
    void onOfferUpdated(Offer updatedOffer)
```

This are some best practises that are already established in other languages like Angular, which we can learn from.

Reply:
see below